implementations/elixir/ockam/ockam_healthcheck/test/healthcheck/healthcheck_test.exs

```diff
@@ -115,17 +193,17 @@ defmodule Ockam.Healthcheck.Test do
       :telemetry.detach("test_handler")
     end)
 
-    assert {:error, :timeout} = Ockam.Healthcheck.check_target(target, 1000)
+    assert {:error, _reason} = Ockam.Healthcheck.check_target(target, 1000)
 
     assert_receive {:telemetry_event, [:ockam, :healthcheck, :result], %{status: 0},
                     %{target: %{name: "target"}}},
-                   1000
+                   5000
 
-    assert_receive {:telemetry_event, [:ockam, :healthcheck, :error], %{duration: _duration},
+    refute_receive {:telemetry_event, [:ockam, :healthcheck, :ok], %{duration: _duration},
```

Comment:
Optional:
```suggestion
    refute_received {:telemetry_event, [:ockam, :healthcheck, :ok], %{duration: _duration},
```

The present-tense `refute_receive` will wait the whole timeout period to ensure no message arrives.

Reply:
Changed in https://github.com/build-trust/ockam/pull/5078/commits/8c703e1753836bf7e958231e9cd8341a7b15b149